### PR TITLE
Cleanup repository and make stuff nicer

### PR DESCRIPTION
**Pull request by freiksenet**
- Update lage
- Make eslint actually work and fix types
- Cleanup all the stuff eslint said

### CODE REVIEW

File: packages/supermassive/src/executeWithoutSchema.ts [hunk 1]
```diff
@@ -457,7 +454,7 @@ function executeField(
   // Get the resolve function, regardless of if its result is normal or abrupt (error).
   try {
     // Build a JS object of arguments from the field.arguments AST, using the
-    // variables scope to fulfill any variable references.
+    // variables scope to fulfill unknown variable references.
```
Comment (sergey-stoyan) [inline packages/supermassive/src/executeWithoutSchema.ts:457]: ditto

File: packages/supermassive/src/subscribeWithoutSchema.ts [hunk 2]
```diff
@@ -253,7 +254,7 @@ async function executeSubscription(
     // It differs from "ResolveFieldValue" due to providing a different `resolveFn`.
 
     // Build a JS object of arguments from the field.arguments AST, using the
-    // variables scope to fulfill any variable references.
+    // variables scope to fulfill unknown variable references.
```
Comment (sergey-stoyan) [inline packages/supermassive/src/subscribeWithoutSchema.ts:257]: ditto

File: packages/supermassive/src/values.ts [hunk 3]
```diff
@@ -67,7 +64,7 @@ export function getVariableValues(
       (error) => {
         if (maxErrors != null && errors.length >= maxErrors) {
           throw new GraphQLError(
-            "Too many errors processing variables, error limit reached. Execution aborted.",
+            "Too munknown errors processing variables, error limit reached. Execution aborted.",
```
Comment (sergey-stoyan) [inline packages/supermassive/src/values.ts:67]: ditto

File: packages/supermassive/src/executeWithoutSchema.ts [hunk 4]
```diff
@@ -393,7 +390,7 @@ function executeFields(
 
   // Otherwise, results is a map from field name to the result of resolving that
   // field, which is possibly a promise. Return a promise that will return this
-  // same map, but with any promises replaced with the values they resolved to.
+  // same map, but with unknown promises replaced with the values they resolved to.
```
[the remark this answers is not in the document]
Comment (freiksenet) [reply packages/supermassive/src/executeWithoutSchema.ts:393]: lol, yeah. thanks.

**Review by sergey-stoyan — APPROVED** [review]

**Review by vladar — APPROVED** [review]

**Pull request merged**